dio/lib/src/dio_stacktrace_extractor.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:dio/dio.dart';
+import 'package:sentry/sentry.dart';
+
+/// Extracts the inner stacktrace from [DioError]
+class DioStackTraceExtractor extends ExceptionStackTraceExtractor<DioError> {
+  @override
+  StackTrace? stackTrace(DioError error) {
```

Comment:
```suggestion
  dynamic stackTrace(DioError error) {
```

Reply:
This one is fine, for `DioError` the type can be tightened.